tests/test__matcher.py

```diff
@@ -350,24 +297,17 @@ def test__main(
     input_file,
     compare_file,
     comparison_type,
-    result_dir,
     output_file,
 ):
     mock_define_output_filename.return_value = "r_file"
     mock_read_csv.return_value = "fake_data"
-    main(input_file, compare_file, result_dir, output_file, comparison_type)
-    if result_dir:
-        mock_mkdir.assert_called_once_with(exist_ok=True)
-    else:
-        mock_mkdir.assert_not_called()
+    main(input_file, compare_file, output_file, comparison_type)
```

Comment:
the intermediate result directory doesn't make sense in a web application. I'm going to move this script out of here so you can call it like a normal script. If you want the result directory with it, I can add it back in.